createlinks

```diff
@@ -28,23 +25,7 @@ event_actions($event,
     'nethserver-fail2ban-shorewall-event' => '90');
 templates2events('/etc/fail2ban/action.d/sendmail-common.local', $event);
 
-# expand urbackup filter
-foreach my $event (qw (
-                        nethserver-urbackup-save
-                        nethserver-urbackup-update
-                    )) {
-                      templates2events('/etc/fail2ban/filter.d/urbackup-auth.conf', $event);
-}
-
 # Fail2ban services restart
-foreach my $event (qw ( nethserver-firewall-base-save
-                        nethserver-urbackup-save
-                        nethserver-urbackup-update
-                    )) {
+foreach my $event (qw ( nethserver-firewall-base-save )) {
     event_services($event,'fail2ban'=> 'restart');
 }
-
-# trusted-networks-modify
-my $event = 'trusted-networks-modify';
-event_services($event, 'fail2ban' => 'restart');
-templates2events('/etc/fail2ban/jail.local', $event);
```

Comment:
some interrogations here, I used to whitelist all trusted networks in fail2ban, to avoid to ban an user connected via vpn or any network we set as safe.
Why did you make this choice ?

Reply:
It's just duplicated code.

When the `trusted-networks-modify` event is called, it calls also the `nethserver-firewall-base` event. The `nethserver-firewall-base`  already does:
- expansion of /etc/fail2ban/jail.local
- restart of fail2ban

Reply:
ok, thank for the answer